en/pro/install.md

```diff
@@ -0,0 +1,87 @@
+% Install Rundeck Pro
```

Comment:
Reading this again, I realize that this is starting to become a duplicate of the administrator's guide, which I wanted to avoid because there's a lot in common with the OSS install guide. This pro page was meant to be a high level overview of the pro features, but the actual installation and configuration should be documented in the administrators guide in the Cluster section. Do you mind integrating your docs here: https://github.com/rundeck/docs/tree/3.0.x/en/administration/cluster ?

Reply:
yes go ahead.

Reply:
Yeah I do think it's good to have the full install explained for cluster. It can probably go here: https://github.com/rundeck/docs/blob/3.0.x/en/administration/cluster/index.md for the administrators to read.

And then the top level pro page can link to it. That way it can stay as a high level description of why you might want pro without diving into the installation details.